src/util/Websocket.ts

```diff
@@ -75,32 +75,37 @@ export default class WS {
             connectionType: "websocket",
         })
     }
+
+    close() {
+        if (this.ws) this.ws.close()
+    }
+    
     debug() {
```

Comment:
to get rid of all the logging, it might be more efficient to simply comment out `this.debug()` on line 36 rather than manually commenting out every log call in the debug function (and leaving a bunch of empty event handlers on the WS object)